python/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -269,7 +269,7 @@ def generateDiracInput(app):
     new_df = uploadLocalFile(job, os.path.basename(compressed_file), tmp_dir)
 
     app.uploadedInput = new_df
-
+    app.is_prepared.associated_files.append(DiracFile(lfn = new_df.lfn))
```

Comment:
Does this deprecate the `app.uploadedInput` now?
I think I'd say go for merge on this PR and worry about removing the `uploadedInput` later if it's duplicate into, I've no idea where or how extensively it's used.